openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -597,6 +618,341 @@ Location: https://client.example.net/cb?
+       |                              |                        |
+       |                              |                        |
+       |  (7) Credential Request      |                        |
+       |      (Access Token, proof(s))|                        |
+       |------------------------------------------------------>|
+       |                              |                        |
+       |  (8) Credential Response     |                        |
+       |       with Credential(s) OR  |                        |
+       |       Transaction ID         |                        |
+       |<------------------------------------------------------|
+~~~
+!---
+Figure: Issuance using the Interactive Authorization Endpoint
+
+## Interactive Authorization Request {#interactive-authorization-request}
+
+All communication with the Interactive Authorization Endpoint MUST utilize TLS.
+
+Note: In case a Wallet Attestation is required by the Authorization Server, it has to be included in this request.
+
+### Initial Request
+
+The initial request to the Interactive Authorization Endpoint is formed and sent in the same way as PAR request as defined in [@!RFC9126, Section 2.1]. The contents of the request are the same as in a regular Authorization Request as defined in (#credential-authz-request), with the following addition:
+
+`interaction_types_supported`: REQUIRED. Comma-separated list of strings indicating the types of interactions that the Authorization Server supports. The order of the values is significant. The following values are defined by this specification:
+
+* `openid4vp_presentation`: Indicates that the Authorization Server supports an OpenID4VP Presentation interaction, as defined in (#iar-require-presentation).
+* `redirect_to_web`: Indicates that the Authorization Server supports a redirect to a web-based interaction, as defined in (#iar-redirect-to-web).
+
+Custom interaction types (see (#iar-custom-extensions)) MAY be defined by the Authorization Server and used in the `interaction_types_supported` parameter.
+
+The following non-normative example shows an initial request to the Interactive Authorization Endpoint:
+
+```http
+POST /iar HTTP/1.1
+Host: server.example.com
+Content-Type: application/x-www-form-urlencoded
+
+response_type=code
+&client_id=CLIENT1234
+&code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+&code_challenge_method=S256
+&redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+&authorization_details=...
+&interaction_types_supported=openid4vp_presentation,redirect_to_web
+```
+
+### Follow-up Request
+
+Follow-up requests to the Interactive Authorization Endpoint only MUST include the `auth_session` value received most recently from the Authorization Server (see (#iar-interaction-required-response)).
+
+Besides `auth_session`, follow-up requests only include the parameters that are relevant for the next step in the authorization process. The specific parameters are defined by each interaction type.
```

Comment:
I don't think "parameters that are relevant for the next step" is correct, the parameters you're including are the ones the authorization server requested for the current step.

```suggestion
Besides `auth_session`, follow-up requests only include the parameters that are in response to the interaction type the Authorization Server requested in the most recent response. The specific parameters are defined by each interaction type.
```